tests/testfunctions/gradlew.bat

```diff
@@ -0,0 +1,104 @@
+@rem
```

Comment:
Since the other build scripts are assuming the user is on  unix system, we don't need to support .bat which is windows specific. I know that this file is autogenerated, but if it ever gives us trouble we should delete it.